compiler/resolution/functionResolution.cpp

```diff
@@ -469,15 +471,19 @@ resolveUninsertedCall(BlockStmt* insideBlock,
   INT_ASSERT(block->parentSymbol);
 
   block->insertAtHead(call);
-  resolveCall(call);
+  if (checkonly && !call->primitive) {
+    resolveNormalCall(call, checkonly);
+  } else {
+    resolveCall(call);
```

Comment:
Commit with this change is here: https://github.com/lydia-duncan/chapel/commit/d1f1a46948765d13f3670cbe3a944bea575703f4